catalogue/webapp/pages/search/images.tsx

```diff
@@ -138,36 +154,38 @@ const ImagesSearchPage: NextPageWithLayout<Props> = ({
         />
       </div>
 
-      <Wrapper>
-        {images?.results && images.totalResults > 0 && (
-          <div className="container">
-            <PaginationWrapper>
-              {images.totalResults > 0 && (
-                <span>{`${images.totalResults} result${
-                  images.totalResults > 1 ? 's' : ''
-                }`}</span>
-              )}
-              <SearchPagination totalPages={images?.totalPages} darkBg />
-            </PaginationWrapper>
-
-            <main>
-              <ImageEndpointSearchResults images={images} />
-            </main>
-
-            <BottomPaginationWrapper>
-              <SearchPagination totalPages={images?.totalPages} darkBg />
-            </BottomPaginationWrapper>
-          </div>
-        )}
+      {images && (
+        <>
+          {images?.totalResults === 0 ? (
+            <SearchNoResults
+              query={queryString}
+              hasFilters={hasFilters({
+                filters: filters.map(f => f.id),
+                queryParams: Object.keys(query).map(p => p),
+              })}
+            />
+          ) : (
+            <Wrapper>
+              <div className="container">
+                <PaginationWrapper>
+                  <span>{`${images.totalResults} result${
+                    images.totalResults > 1 ? 's' : ''
+                  }`}</span>
```

Comment:
Let's look at styling this consistently with TotalResultsCopy in stories.tsx, and possibly pushing down the pluralisation copy into the component as well.